include/OpenSpaceToolkit/Physics/Coordinate/Spherical/LLA.hpp

```diff
@@ -92,6 +154,72 @@ class LLA
         const Real& anEllipsoidFlattening
     );
 
+    /// @brief                  Calculate the azimuth angles between two LLA coordinates
+    ///
+    /// @param                  [in] aFirstLLA A first LLA coordinate
+    /// @param                  [in] aSecondLLA A second LLA coordinate
+    /// @param                  [in] anEllipsoidEquatorialRadius An ellipsoid equatorial radius
+    /// @param                  [in] anEllipsoidFlattening An ellipsoid flattening
+    /// @return                 Azimuth angles between the two LLA coordinates
+
+    static Pair<Angle, Angle> AzimuthBetween(
+        const LLA& aFirstLLA,
+        const LLA& aSecondLLA,
+        const Length& anEllipsoidEquatorialRadius,
+        const Real& anEllipsoidFlattening
+    );
+
+    /// @brief                  Calculate a point between two LLA coordinates
+    ///
+    /// @param                  [in] aFirstLLA A first LLA coordinate
+    /// @param                  [in] aSecondLLA A second LLA coordinate
+    /// @param                  [in] aRatio A ratio
+    /// @param                  [in] anEllipsoidEquatorialRadius An ellipsoid equatorial radius
+    /// @param                  [in] anEllipsoidFlattening An ellipsoid flattening
+    /// @return                 A point between the two LLA coordinates
+
+    static LLA IntermediateBetween(
+        const LLA& aFirstLLA,
+        const LLA& aSecondLLA,
+        const Real& aRatio,
+        const Length& anEllipsoidEquatorialRadius,
+        const Real& anEllipsoidFlattening
+    );
+
+    /// @brief                  Propagate LLA coordinates in provided direction and distance
+    ///
+    /// @param                  [in] aLLA A LLA coordinate
+    /// @param                  [in] aDirection An azimuth angle
+    /// @param                  [in] aDistance A distance
+    /// @param                  [in] anEllipsoidEquatorialRadius An ellipsoid equatorial radius
+    /// @param                  [in] anEllipsoidFlattening An ellipsoid flattening
+    /// @return                 Propagated LLA coordinate
+
+    static LLA Forward(
+        const LLA& aLLA,
+        const Angle& aDirection,
+        const Length& aDistance,
+        const Length& anEllipsoidEquatorialRadius,
+        const Real& anEllipsoidFlattening
+    );
+
+    /// @brief                  Generate equidistant intermediate LLAs between two LLA coordinates.
+    ///
+    /// @param                  [in] aFirstLLA A first LLA coordinate
+    /// @param                  [in] aSecondLLA A second LLA coordinate
+    /// @param                  [in] aNumberOfPoints A number of points
+    /// @param                  [in] anEllipsoidEquatorialRadius An ellipsoid equatorial radius
+    /// @param                  [in] anEllipsoidFlattening An ellipsoid flattening
+    /// @return                 intermediate equidistant LLAs between the two LLA coordinates
+
+    static Array<LLA> IntermediateLLAs(
```

Comment:
For the naming, what about 
`IntermediateLLAsBetween`

Reply:
Ooooh. How about `Linspace`. That's a popularly used function in other libraries that most people are familiar with.

Reply:
Yeah I like that, good idea. I still think you should be consistent with the naming between the regular methods and static methods of each kind, cause there are a few discrepancies in their naming